naplib/localization/freesurfer.py

```diff
@@ -1295,6 +1248,32 @@ def interpolate_electrodes_onto_brain(self, coords, values, isleft=None, k=10, m
         self.rh.interpolate_electrodes_onto_brain(coords[~isleft], values[~isleft], k=k, max_dist=max_dist, roi=roi)
         return self
 
+    def parcellate_overlay(self, merge_func=np.mean):
+        """Merges brain overlay values within each atlas parcel.
+
+        This method applies a merging function to the overlay values within each
+        anatomical parcel defined by an atlas.  It is typically used after
+        interpolating electrode data onto the brain surface
+        (e.g., via `brain.interpolate_electrodes_onto_brain()`) to summarize
+        the data within each parcel.
+
+        Parameters
+        ----------
+        merge_func : callable, optional
```

Comment:
I think it's better to say "callable, default=numpy.mean", because optional means that you can pass None and it will be fine, which is not the case